src/llmcompressor/modifiers/quantization/quantization/mixin.py

```diff
@@ -145,6 +152,10 @@ def resolved_targets(self) -> Set[str]:
         for config_group in self.resolved_config.config_groups.values():
             for target in config_group.targets:
                 targets.add(target)
+
+        if self.resolved_config.kv_cache_scheme is not None:
+            targets.add("re:.*self_attn$")
```

Comment:
I thought we didn't want to do this to find target attention modules, instead looping over `model.modules()` and checking `is_attention(module)` as the true check?

Reply:
I decided to leave decoupling `"re:.*self_attn$"` and `is_attention_module` for a later PR.

Reply:
i think it'd be worth leaving a todo or note in that case